src/components/Note/Note.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import { Note, TextLink } from '@contentful/forma-36-react-components';
+
+export interface INoteProps {
+  type: 'primary' | 'positive' | 'negative' | 'warning';
+  error: Error;
+  resetErrorBoundary: () => void;
+}
+
+export function IxNote({
+  error,
+  type,
+  resetErrorBoundary,
+}: INoteProps): ReactElement {
+  const [message, _link, ...rest] = error.message.split('$');
+  const [link, title] = _link.split('|');
+  const linkTitle = title?.length ? title : link;
+  console.log(message, link, ...rest);
+
+  return (
+    <div className="ix-note" style={{ marginTop: 25 }}>
+      <Note
+        noteType={type}
+        title={error.name}
+        hasCloseButton
+        onClose={resetErrorBoundary}
+      >
+        {message + ' '}{' '}
+        <TextLink target="window" href={link}>
+          {linkTitle}
+        </TextLink>{' '}
+        {' ' + rest}
```

Comment:
I'm little confused, what's with all the `{' '}`?

Reply:
🤔  yea not sure what I was going for here. Will remove the `{' '}` that aren't needed.